commit 053aa9e09f69dfe3bffc2d4519594c83af32d02d (refs/changes/28/3728/1)
Author: Dave Eddy <dave@daveeddy.com>
Date:   2018-03-27T15:43:51-04:00 (1 year, 6 months ago)
    
    TRITON-279 vms.full.test find_fabric_vm_package implemented incorrectly and not needed

diff --git a/test/vms.full.test.js b/test/vms.full.test.js
index 04fcf87..46be1f4 100644
--- a/test/vms.full.test.js
+++ b/test/vms.full.test.js
@@ -2740,16 +2740,6 @@ exports.ensure_no_fabric_nat_provisioned = function (t) {
 };
 
 
-exports.find_fabric_vm_package = function (t) {
-    fabricNetwork = NETWORKS.find(function _findFabricNetwork(n) {
-        return n.fabric === true;
-    });
-
-    t.ok(fabricNetwork, 'found a fabric network');
-    t.done();
-};
-
-
 exports.create_vm_on_fabric_network = function (t) {
     if (!fabricNetwork) {
         t.fail('No fabric network available');
